docroot/sites/all/modules/features/forms2/forms2.install

```diff
@@ -0,0 +1,11 @@
+**
+ * Removes field_yes_no from scheinman_webcast_registration form and two other
+ * fields altogether
+ */
+function ilr_forms2_update_7001() {
```

Comment:
Hooks in Drupal are structured as [module_name]_[hook_name]_[other_info]. 

So, for hook_update_n (see https://api.drupal.org/api/drupal/modules%21system%21system.api.php/function/hook_update_N/7), the module name here is "forms2", not "ilr_forms2". 

Also, it's not clear to me at this point why Ted was running both field_delete_field and field_delete_instance, as the former calls the latter. Please redo this commit and force push. Thanks.